server/backends/invocationdb/invocationdb.go

```diff
@@ -58,17 +57,12 @@ func (d *InvocationDB) InsertOrUpdateInvocation(ctx context.Context, ti *tables.
 	return d.h.Transaction(ctx, func(tx *db.DB) error {
 		var existing tables.Invocation
 		if err := tx.Where("invocation_id = ?", ti.InvocationID).First(&existing).Error; err != nil {
-			if db.IsRecordNotFound(err) {
-				return d.createInvocation(tx, ctx, ti)
-			}
-		} else {
-			err := tx.Model(&existing).Where("invocation_id = ?", ti.InvocationID).Updates(ti).Error
-			if err != nil {
-				log.Warningf("Error updating invocation %s: %s", ti.InvocationID, err.Error())
```

Comment:
are we still logging the invocationID and err in another place?

Reply:
Yep, it manifests in the logs like this:

```
2021/11/04 13:45:46.290 WRN Disconnecting invocation "5eaa482a-5c4a-449e-84cc-d70fe02e1462": context deadline exceeded
...
2021/11/04 13:45:46.290 WRN Error finalizing invocation "5eaa482a-5c4a-449e-84cc-d70fe02e1462" during disconnect: context deadline exceeded
```